modeltranslation/fields.py

```diff
@@ -256,6 +258,17 @@ def __ne__(self, other):
     def __hash__(self):
         return hash((self.creation_counter, self.language))
 
+    def get_default(self):
```

Comment:
suggestion: With this approach, we need to force evaluation in two places, here and in `deconstruct`.

Maybe we can do this just once in `__init__`?

Reply:
that would break the behaviour of actual callable `default=some_function`, as the `__init__` of the field is only invoked once (similarly to how `models.CharField(...)` itself executes only once) when the translation field instance is created and added to the model.

E.g. if you were to specify `default=timezone.now`, this would effectively be the same thing as setting `default=timezone.now()` which is *very* different behaviour.